src/main/java/com/owncloud/android/lib/common/OwnCloudAccount.java

```diff
@@ -135,4 +139,42 @@ public String getDisplayName() {
             return null;
         }
     }
+
+    /*
+     * Autogenerated Parcelable interface
+     */
+
+    @Override
+    public int describeContents() {
+        return 0;
+    }
+
+    @Override
+    public void writeToParcel(Parcel dest, int flags) {
+        dest.writeParcelable(this.baseUri, flags);
```

Comment:
is "this" needed?
We have this in no other place in our code and it makes it harder to read.

Reply:
@tobiasKaminsky 

It is not neede per se, but it is autogenerated code and that is the kind of template Android Studio plugin uses.

I removed it - not everybody will use this particular plugin so we can assume it will be modified manually.